less/window.less

```diff
@@ -83,6 +83,7 @@ html,
     height: 100%;
     width: 100%;
     &.isPreview {
+      will-change: opacity;
```

Comment:
yes the browser reserves a chunk of memory for that so they're costly if done several times but new tab load and tab's loading animation can benefit too. lmk if you know some other areas. it applies for `opacity` and `transform` properties.